minicluster/src/main/java/alluxio/multi/process/MultiProcessCluster.java

```diff
@@ -162,16 +162,16 @@ public synchronized void start() throws Exception {
     mState = State.STARTED;
 
     mMasterAddresses = generateMasterAddresses(mNumMasters);
+    MasterNetAddress masterAddress = mMasterAddresses.get(0);
```

Comment:
why do we move this outside? I think we only need it for `UFS_NON_HA`

Reply:
Moved back to `UFS_NON_HA`